provision.sh

```diff
@@ -287,7 +287,7 @@ MAKE_ARGS="local_requirements prepare_host network install_stack"
 if [[ $CLUSTER_NAME != *"az"* ]]; then
     MAKE_ARGS="${MAKE_ARGS} prepare_stack local_os_client"
 fi
-make $MAKE_ARGS
+make $MAKE_ARGS post_install
```

Comment:
Just seeing the diff with `make $MAKE_ARGS post_install` got me confused and I though you were only running the post-install target. I suggest you write as follow to make it a (little bit) clearer:

```suggestion
MAKE_ARGS="${MAKE_ARGS} post_install"
make $MAKE_ARGS
```

You could also rename `MAKE_ARGS` to `MAKE_TARGETS` for extra clarity.